src/box/session_settings.c

```diff
@@ -49,6 +49,7 @@ const char *session_setting_strs[SESSION_SETTING_COUNT] = {
 	"sql_recursive_triggers",
 	"sql_reverse_unordered_selects",
 	"sql_select_debug",
+	"sql_seq_scan",
```

Comment:
I don't see any information about this session setting key in the doc bot request.

Reply:
Thank you for the review! Added.